gatsby-node.ts

```diff
@@ -0,0 +1,36 @@
+import { createFilePath } from 'gatsby-source-filesystem';
+import { GatsbyNode } from 'gatsby';
+
+export const onCreateNode: GatsbyNode['onCreateNode'] = ({
```

Comment:
All exports should be at the end of the file, this should be fixed elsewhere else
```suggestion
const onCreateNode: GatsbyNode['onCreateNode'] = ({
***
export { onCreateNode }
```